File: counting/count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        unordered_map<int,int> mp;
        for(int x : nums) {
            mp[x]++;
        }
        int mx = 0;
        for(auto p : mp) {
            mx = max(mx, p.second);
        }
        int ans = 0;
        for(auto p : mp) {
            if(p.second == mx) ans += p.second;
        }

        return ans;
    }
};

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int[] freq=new int[101];
        int max=-1;
        for(int i:nums){
            freq[i]++;
            max=Math.max(max,freq[i]);
        }
        int sum=0;
        for(int i=0;i<=100;i++){
            if(freq[i]==max){
                sum+=max;
            }
        }
        return sum;
    }
}